src/main/java/javax/resource/spi/work/ExecutionContext.java

```diff
@@ -37,10 +37,10 @@
  * (in the future) without forcing resource adapter implementations 
  * to change.</ul>
  *
- * Note: Resource adapters that are developed for Connectors 1.6 specification
- * compliant application servers and above, are recommended to use
+ * Note: Resource adapters that are developed for Jakarta Connectors specification
+ * compliant application servers, are recommended to use
  * the <code>TransactionContext</code> interface instead of this 
- * class. See Chapter.11 Generic Work Context in the Connectors 1.6
+ * class. See Chapter.11 Generic Work Context in the Java&trade; Connectors 1.6
```

Comment:
This should be `JavaEE&trade;` rather than `Java&trade;` because it is referring to the JCA 1.6 spec document.

Reply:
It should be "Jakarta Connectors".

"`JavaEE&trade;`" is not correct ever.

Reply:
As it refers to an old version shouldn't it remain Java connectors?

Reply:
Put back original wording